packages/cli/e2e-test/helpers.js

```diff
@@ -117,12 +117,29 @@ async function waitForPageWithText(
     }
   }
 
+  // The page may not be fully loaded and hence we need to retry.
+  const maxFindTextAttempts = 3;
```

Comment:
this should be put along with `maxLoadAttempts = 240` as a parameter at the top, right?

Reply:
Thanks @freben ! I think so too.